src/components/playback/playbackmanager.js

```diff
@@ -1848,6 +1848,31 @@ class PlaybackManager {
                     SortBy: options.shuffle ? 'Random' : 'SortName',
                     MediaTypes: 'Audio'
                 });
+            } else if (firstItem.Type === 'Series') {
+                promise = new Promise(function (resolve, reject){
+                    const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
+                    apiClient.getEpisodes(firstItem.Id, {
```

Comment:
Like this. I'm typing on my phone, so I can't check.
```suggestion
                const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
                promise = apiClient.getEpisodes(firstItem.Id, {
```